addons/website_slides/static/src/js/slides_course_slides_list.js

```diff
@@ -77,9 +58,22 @@ publicWidget.registry.websiteSlidesCourseSlidesList = publicWidget.Widget.extend
                 model: "slide.slide",
                 ids: self._getSlides()
             }
+        }).then(function (res) {
+            self._toggleCategoryEmptyFlag();
         });
     },
-
+    _toggleCategoryEmptyFlag: function (){
+        var categories = $('.o_wslides_slide_list');
+        for (var i = 0; i < categories.length; i++){
```

Comment:
You could simplify by using:

`this.$('.o_wslides_slide_list').each(function () {...});`